### ISSUE TITLE
Gradient option is not working in the child theme

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/tiny-hestia#14]
because of this line 
`wp_deregister_style('hestia_style');`
in functions.php.

https://wordpress.org/themes/tiny-hestia/

@cristian-ungureanu can you please check what the best way would be to fix this problem?

**Comment by cristian-ungureanu** [comment]
@rodica-andronache I think it shouldn't work, That was the main idea of this child theme, to keep it clean without additional css or js. When I implemented it there was a request to clean the css, so I removed the one form parent and generate a clean one. 
I can generate it again and update the child theme ( but this is a must every time we add something new ) or I can remove its css and keep the one from parent ( but then it won't be as clean as it is now ).
